content/issue-2/optimize-rust-comptime.md

```diff
@@ -0,0 +1,200 @@
+
+Although there are often complaints saying Rust'compilation speed is notorious slow, our project [RisingWave](https://github.com/risingwavelabs/risingwave) is not very slow to compile, especially since previously contributors like ([skyzh](https://github.com/skyzh), [BugenZhao](https://github.com/bugenzhao)) have put in a lot of effort. After using an M1 MacBook Pro, compiling is not a problem at all. A full debug compilation only takes 2-3 minutes.
+
+However, over time, more and more things have been added to our CI, making it increasingly bloated. The main workflow now takes about 40 minutes, while the PR workflow takes about 25 minutes 30 seconds. Although it is still not intolerably slow, it is already noticeably slower than before.
+
+So a few days ago, I decided to spend some time researching whether I could optimize the compilation speed a bit more.
+
+What shocked me was that there were some very simple methods that, with just a little effort, produced astonishing results. I feel like I can describe them as low-hanging fruits, silver bullets, or even free lunch 🤯.
+
+---
+
+P.S. I highly recommend [matklad](https://github.com/matklad)'s blog (He is the original author of IntelliJ Rust and rust-analyzer):
+- [Fast Rust Builds](https://matklad.github.io/2021/09/04/fast-rust-builds.html)
+- [Delete Cargo Integration Tests](https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html)
+
+Most of the methods I used are discussed here, and he explains them clearly. If not otherwise indicated, all quotes in this article come from here.
```

Comment:
I believe we mean `there` instead of `here` :)
```suggestion
Most of the methods I used are discussed there, and he explains them clearly. If not otherwise indicated, all quotes in this article come from there.
```